website/project/model.py

```diff
@@ -1313,6 +1342,13 @@ def nodes_primary(self):
             if node.primary
         ]
 
+    def node_and_primary_descendants(self):
+        """Gets an iterator for a node and all of its visible descendants
```

Comment:
This docstring say "visible", but the function name says "primary". Is there a distinction?